ui/articles-aem-renderer/src/androidTest/java/org/godtools/articles/ExampleInstrumentedTest.java

```diff
@@ -0,0 +1,90 @@
+package org.godtools.articles;
+
+import android.support.test.runner.AndroidJUnit4;
+
+import org.godtools.articles.aem.db.ArticleRepository;
+import org.godtools.articles.aem.db.ArticleRoomDatabase;
+import org.godtools.articles.aem.db.AttachmentRepository;
+import org.godtools.articles.aem.db.ManifestAssociationRepository;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * Instrumented test, which will execute on an Android device.
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+@RunWith(AndroidJUnit4.class)
+public class ExampleInstrumentedTest {
+
+    private ArticleRepository mArticleDao;
+    private AttachmentRepository mAttachmentDao;
+    private ManifestAssociationRepository mAssociationDao;
+    private ArticleRoomDatabase db;
+
+
+    @Before
+    public void createDb() {
+//        Context context = InstrumentationRegistry.getTargetContext();
```

Comment:
did the room db tests just not work out?

Reply:
I had the DAOs as public when I created the test.  But I wanted to make them Package Protected and Use the Repositories to access the data.  I didn't need to test the Repositories right now because they are just calling the DAO.  Later the repository may contain more logic for interacting with the database.

Reply:
you can still use the tests even if the DAOs are package private, just put them in the same package :)